--- vendor/cleantalk/yii2-antispam/Component.php	2024-08-07 22:32:58
+++ orignial_patch/Component.php	2024-08-07 23:05:17
@@ -1,9 +1,8 @@
 <?php
 namespace cleantalk\antispam;
 
-use Cleantalk\Cleantalk;
-use Cleantalk\CleantalkRequest;
-use Cleantalk\CleantalkResponse;
+use Cleantalk;
+use CleantalkRequest;
 use InvalidArgumentException;
 use Yii;
 use yii\base\Component as BaseComponent;
\ No newline at end of file
@@ -135,7 +134,7 @@
 
     /**
      * Create request for CleanTalk API.
-     * @return CleantalkRequest
+     * @return \CleantalkRequest
      */
     protected function createRequest()
     {
\ No newline at end of file
@@ -156,9 +155,9 @@
     }
 
     /**
-     * @param CleantalkRequest $request
+     * @param \CleantalkRequest $request
      * @param string $method
-     * @return CleantalkResponse CleanTalk API call result
+     * @return \CleantalkResponse CleanTalk API call result
      * @throws InvalidArgumentException
      */
     protected function sendRequest($request, $method)
\ No newline at end of file
