--- vendor/cleantalk/yii2-antispam/Component.php	2024-08-07 23:11:26
+++ orignial_patch/Component.php	2024-08-07 23:10:47
@@ -27,7 +27,7 @@
     public $apiKey;
 
     /** @var string API URL */
-    public $apiUrl = 'http://moderate.cleantalk.ru';
+    public $apiUrl = 'http://moderate.cleantalk.org';
 
     /**
      * @deprecated Use setting in https://cleantalk.org/my/service?action=edit
\ No newline at end of file
